src/views/tiles/NodeView.js

```diff
@@ -190,7 +191,7 @@ exports = Class(ImageView, function (supr) {
 				}
 			}
 
-			if (tile.id && node.characterSettings && tile.tags && !tile.tags.locked) {
+			if (tile.id && node.characterSettings && tile.tags) {
```

Comment:
will show numbers on locked tiles too.

If the numbers have to be hidden, the characterSettings popup shouldn't be set